jenkins_build.sh

```diff
@@ -10,5 +10,6 @@ easy_install nose
 easy_install mock
 easy_install requests
 easy_install coverage
+easy_install pytest
```

Comment:
I am not sure where this file is used in this library. Perhaps I need to remove it or change it so it will not use `easy_install` nor use `nose` for tests.